### PR TITLE
updated x64 and ARM64 instructions

### PR DESCRIPTION
**Pull request by hengm3467**
I updated the chipset-related instructions based on the discussion. Please let me know if these are accurate.

### CODE REVIEW

File: docs/Get-Started.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You can run RisingWave in these ways:
 
 ### Install and run from a Docker image (Linux & macOS)
 
-You can install and run RisingWave from a Docker image. Currently, only x86-64 platforms are supported.
+You can install and run RisingWave from a Docker image on x64 systems. Images for ARM64 systems (including macOS devices with Apple M1 chips) might be available for testing purpose, but it is not guaranteed.
```
Comment (skyzh) [inline docs/Get-Started.md]: x64 -> x86-64 or amd64

**Review by skyzh — APPROVED** [review]
Rest LGTM

File: docs/Get-Started.md (hunk 1 above)
Comment (hengm3467) [reply docs/Get-Started.md]: @skyzh As I understand it, x64 covers both x86_64 and amd64. Do we want to single either of them out?

Comment (skyzh) [reply docs/Get-Started.md]: I think either of x86-64 and amd64 would be okay here. I seldom see x64...

Comment (hengm3467) [reply docs/Get-Started.md:52]: or we can keep referring to it as x86-64.

Comment (hengm3467) [reply docs/Get-Started.md]: > I think either of x86-64 and amd64 would be okay here. I seldom see x64...

Cool. I changed it to AMD64.

**Pull request merged**